README.md

```diff
@@ -1,20 +1,31 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/meshery-kuma/master/img/readme/meshery-logo-light-text.svg"  width="70%" /></a><br /><br /></p>
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-kuma/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
```

Comment:
needs a fix , the specified location does not exist !!

Reply:
@legendarykamal, well.. The specified location wont even exist if the PR is not merged. Because in this commit. I have added the images in .github/assets/images/meshery directory. And it is trying to find the image in this directory in master branch. But this directory doesn't exist in master branch as of now. After merging, this path will become valid and you will be able to see the logo.
I have also tested this in my forked repository. When I change the username and the branch name, it starts showing the image.